File: fix_tabs.js
const fs = require('fs');
let html = fs.readFileSync('turpone-products/index.html', 'utf8');

// 1. Change menu link in header
html = html.replace(/>Turpone Products</g, '>Food Services<');
html = html.replace(/>Turpone Products </g, '>Food Services <');

// 2. Delete Retail Tab Button
html = html.replace(/<button aria-controls="e-n-tab-content-2183931401"[^>]*>.*?Retail\s*<\/span><\/button>/s, '');

// 3. Delete Retail Tab Content
html = html.replace(/<div aria-labelledby="e-n-tab-title-2183931401".*?(?=<div aria-labelledby="e-n-tab-title-2183931402")/s, '');

// 4. Make Food Service tab active
html = html.replace('aria-controls="e-n-tab-content-2183931402" aria-selected="false"', 'aria-controls="e-n-tab-content-2183931402" aria-selected="true"');
html = html.replace(/(<div aria-labelledby="e-n-tab-title-2183931402"[^>]*class=")([^"]*)/, '-active ');

fs.writeFileSync('turpone-products/index.html', html, 'utf8');

File: fix_tabs.py
import re

with open('turpone-products/index.html', 'r', encoding='utf-8') as f:
    html = f.read()

# 1. Change the menu link in the header:
html = html.replace('>Turpone Products<', '>Food Services<')
html = html.replace('>Turpone Products <', '>Food Services <')

# 2. Delete the Retail Tab Button
html = re.sub(r'<button aria-controls="e-n-tab-content-2183931401"[^>]*>.*?Retail\s*</span></button>', '', html)

# 3. Delete the Retail Tab Content
html = re.sub(r'<div aria-labelledby="e-n-tab-title-2183931401".*?(?=<div aria-labelledby="e-n-tab-title-2183931402")', '', html, flags=re.DOTALL)

# 4. Make the Food Service tab active by default
html = html.replace('aria-controls="e-n-tab-content-2183931402" aria-selected="false"', 'aria-controls="e-n-tab-content-2183931402" aria-selected="true"')
html = re.sub(r'(<div aria-labelledby="e-n-tab-title-2183931402"[^>]*class=")([^"]*)', r'\g<1>e-active \g<2>', html)

with open('turpone-products/index.html', 'w', encoding='utf-8') as f:
    f.write(html)
